clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -911,7 +911,7 @@ void EmitAssemblyHelper::RunOptimizationPipeline(
       PB.registerPipelineStartEPCallback(
           [&](ModulePassManager &MPM, OptimizationLevel Level) {
             MPM.addPass(ESIMDVerifierPass(LangOpts.SYCLESIMDForceStatelessMem));
-            MPM.addPass(SYCLPropagateAspectsUsagePass());
+            MPM.addPass(SYCLPropagateAspectsUsagePass({"fp64"}));
```

Comment:
```suggestion
            MPM.addPass(SYCLPropagateAspectsUsagePass(/*ExcludeAspects =*/{"fp64"}));
```

It was counter-intuitive to me that the first argument passed is what we want excluded, and I think the comment makes it a little clearer.   Could be just me though.

Reply:
I agree, having the comment is a lot more informative. It has been added. 👍